docs/diary/2026-01-30_slug-based-routing.md

```
@@ -0,0 +1,131 @@
+# 2026-01-30 URL 라우팅 slug 전환
+
+## 배경
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)